SPECS/debugedit/debugedit.spec

```
@@ -1,36 +1,44 @@
-Summary:      Debugedit - obtain debug information from binaries.
-Name:         debugedit
-Version:      5.0
-Release:      2%{?dist}
-License:      GPLv3+
-URL:          https://sourceware.org/debugedit/
-Vendor:       Microsoft Corporation
-Distribution: Mariner
-Source0:      https://sourceware.org/ftp/%{name}/%{version}/%{name}-%{version}.tar.xz
+Summary:        Debugedit - obtain debug information from binaries.
+Name:           debugedit
+Version:        5.0
+Release:        3%{?dist}
+License:        GPLv3+
+URL:            https://sourceware.org/debugedit/
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://sourceware.org/ftp/%{name}/%{version}/%{name}-%{version}.tar.xz
+Patch0:         BUG-28161.patch
+BuildRequires:  automake
+BuildRequires:  binutils
+BuildRequires:  help2man
 
 %description
 %{summary}
 
 %prep
-%autosetup
+%autosetup -p1
 
 %build
+autoreconf -fvvi
 %configure
 %make_build
 
 %install
 %make_install
 
 %check
-%make_build_check
+make check || { cat tests/testsuite.log; exit 1; }
```

Please, never use `exit` in the `%check` section. It's breaking the package builds instead of only marking the test as failing.
```suggestion
make check || { cat tests/testsuite.log; false; }
```